src/main/java/teammates/storage/sqlapi/NotificationsDb.java

```diff
@@ -53,7 +56,10 @@ public Notification createNotification(Notification notification)
     public Notification getNotification(UUID notificationId) {
         assert notificationId != null;
 
-        return HibernateUtil.get(Notification.class, notificationId);
+        Notification notif = HibernateUtil.get(Notification.class, notificationId);
+        // HibernateUtil.flushSession();
+        // HibernateUtil.evict(notif);
+        return notif;
```

Comment:
Would prefer to call evict within the db layer instead of in the logic layer. However, this results in a `NullPointerError` caused by `get` returning `null` even though the entity exists in the database. Unsure why this does not happen when evict is called in the logic layer instead. Any help is appreciated!